pmm-app/tests/verifyPMMDemo_test.js

```diff
@@ -0,0 +1,46 @@
+Feature('Verify PMMDemo side');
+
+Scenario('PMM-T363 - Verify Copyrights & Legal section elements', async (I, pmmDemoPage) => {
+  I.amOnPage(pmmDemoPage.url);
+  I.waitForVisible(pmmDemoPage.fields.title, 30);
+  pmmDemoPage.verifyCopyrightsAndLegal();
+  I.amOnPage(pmmDemoPage.url + pmmDemoPage.mongoDBDashbordUrl);
+  pmmDemoPage.verifyCopyrightsAndLegal();
+});
+
+Scenario(
+  'PMM-T364 - Verify PMM settings returns Access denied error',
```

Comment:
@IvanaWzientkova could you please add @not-pr-pipeline tag to each scenario, we don't these tests running for Grafana Dashboard PR's, we will have another job to execute them, also use tag @pmm-demo, so we can group scenarios for pmm-demo together.